packages/flutter_tools/lib/src/build_runner/web_compilation_delegate.dart

```diff
@@ -0,0 +1,229 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: implementation_imports
+import 'package:build/build.dart';
+import 'package:build_modules/build_modules.dart';
+import 'package:build_web_compilers/build_web_compilers.dart';
+import 'package:build_web_compilers/builders.dart';
+import 'package:logging/logging.dart';
+import 'package:meta/meta.dart';
+import 'package:build_runner_core/build_runner_core.dart';
+import 'package:build_config/build_config.dart';
+import 'package:build_modules/builders.dart';
+import 'package:build_runner_core/src/generate/build_impl.dart';
+import 'package:build_runner_core/src/generate/options.dart';
+import 'package:path/path.dart' as path;
+import 'package:watcher/watcher.dart';
+import 'package:build_modules/src/module_builder.dart';
+import 'package:build_modules/src/platform.dart';
+import 'package:build_web_compilers/src/dev_compiler_bootstrap.dart';
+
+import '../artifacts.dart';
+import '../base/file_system.dart';
+import '../base/logger.dart';
+import '../compile.dart';
+import '../dart/package_map.dart';
+import '../globals.dart';
+import '../web/compile.dart';
+
+const String ddcBootstrapExtension = '.dart.bootstrap.js';
+const String jsEntrypointExtension = '.dart.js';
+const String jsEntrypointSourceMapExtension = '.dart.js.map';
+const String jsEntrypointArchiveExtension = '.dart.js.tar.gz';
+const String digestsEntrypointExtension = '.digests';
+const String jsModuleErrorsExtension = '.ddc.js.errors';
+const String jsModuleExtension = '.ddc.js';
+const String jsSourceMapExtension = '.ddc.js.map';
+
+final DartPlatform flutterWebPlatform =
+    DartPlatform.register('flutter_web', <String>[
+  'async',
+  'collection',
+  'convert',
+  'core',
+  'developer',
+  'html',
+  'html_common',
+  'indexed_db',
+  'js',
+  'js_util',
+  'math',
+  'svg',
+  'typed_data',
+  'web_audio',
+  'web_gl',
+  'web_sql',
+  '_internal',
+  // Flutter web specific libraries.
+  'ui',
+  '_engine',
+  'io',
+  'isolate',
+]);
+
+/// The build application to compile a flutter application to the web.
+final List<BuilderApplication> builders = <BuilderApplication>[
+  apply('flutter_tools|module_library',
```

Comment:
A library prefix for whichever import 'apply'  comes from will probably be helpful. (Since there are so many imports here, this comment should probably also be applied to any of the above libraries that have top-levels with common names.)